src/TauBounds.cpp

```diff
@@ -7,11 +7,6 @@
 #include "McRefCommon.h"
 #include "MemoryMng.h"
 
-double *tau_ubounds; // Temporary array. Reinitialized per iteration. Holds upper tau bounds
-double *tau_lbounds; // Ditto, but with lower tau bounds
-int **lca_pops; // "cache" of the lca of every pair of pops. e.g. - lca_pops[1][2] is the lca of pops 1 & 2
-
```

Comment:
where is lca_pops 2D array defined now? I see that the tau_bound arrays are defined in TauBounds.h, but I couldn't find the definition of this array.